app/src/models/types.ts

```diff
@@ -23,3 +24,16 @@ export type ExtendedApplicationData = ApplicationData & {
   email: string
   thresholdExceeded: number
 }
+
+export type ExtendedLoadBalancerData = {
+  [any: string]: {
+    id: string
+    userID: string,
+    name: string,
+    applicationIDs: string[]
+    percentageUsed: number
+    maxRelays: number,
+    relaysUsed: number
+    applicationsRelayed: ApplicationData & { id: string }[]
```

Comment:
I'm not 100% happy with this name—maybe `activeApplications`? This sounds like if something happened to the applications, instead of referring to the applications that are _doing_ something.